js/projects.js

```diff
@@ -0,0 +1,217 @@
+            <li class="project-lang-item">css</li>
+            <li class="project-lang-item">bootstrap</li>
+            <li class="project-lang-item">ruby</li>
+          </ul>
+          <button id="4" class="button project-btn jsBtn">See Project</button>
+        </div>
+
+        <div id="project-5">
+          <h3 class="project-heading">Professional Art Printing Data</h3>
+          <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+          <ul class="project-lang">
+            <li class="project-lang-item">html</li>
+            <li class="project-lang-item">css</li>
+            <li class="project-lang-item">bootstrap</li>
+            <li class="project-lang-item">ruby</li>
+          </ul>
+          <button id="5" class="button project-btn jsBtn">See Project</button>
+        </div>
+
+        <div id="project-6">
+          <h3 class="project-heading">Professional Art Printing Data</h3>
+          <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+          <ul class="project-lang">
+            <li class="project-lang-item">html</li>
+            <li class="project-lang-item">css</li>
+            <li class="project-lang-item">bootstrap</li>
+            <li class="project-lang-item">ruby</li>
+          </ul>
+          <button id="6" class="button project-btn jsBtn">See Project</button>
+        </div>
+      </div>
+    </div>
+`;
+
+// CTREATE POP UP WINDOW
+const prjBtn = document.querySelectorAll('.jsBtn');
+prjBtn.forEach((button) => {
+  button.addEventListener('click', () => {
+    const section = document.createElement('section');
+    section.className = 'section-menu';
+    section.innerHTML = `
+    <div class="div-menu">
+      <h3 id="div-menu-h3">${projects[button.id].title}</h3>
+      <img id="menu-close-btn" src="./images/nav/close.png" />
+    </div>
+    <div class="div-menu">
+      <ul>
+        <li class="div-menu-li">qwerty</li>
+        <li class="div-menu-li">azerty</li>
+        <li class="div-menu-li">qwertz</li>
+      </ul>
+    </div>
+    <div id="div-section">
+      <div class="div-menu-img"><img id="div-section-img" src="${projects[button.id].image}" alt="" /></div>
+      <div id="div-section-menu">
+        <div class="div-menu"><p id="div-section-p">${projects[button.id].description}</p></div>
+        <div id="div-menu-last">
+          <a id="div-menu-live" href="${projects[button.id].live_version}"><button class="div-menu-button">See Live&nbsp;<div><img src="./images/social-media-header/live.png" alt="live icon" /></div></button></a>
+          <a id="div-menu-source" href="${projects[button.id].source_code}"><button class="div-menu-button">See Source&nbsp;<div><img src="./images/social-media-header/github.png" alt="" /></div></button></a>
+        </div>
+      </div>
```

Comment:
- [ ] As stated the project requirement  You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). So Kindly Adjust The popup to make it responsive, to fits desktop sizes properly.

Your Desktop size Popup Look likes this : 
![Capture](https://user-images.githubusercontent.com/93598189/204037425-a675368f-94cb-464f-ba70-50145436b25e.PNG)

The Figma Desktop size Popup Look likes this : 
![Capture](https://user-images.githubusercontent.com/93598189/204037527-7f6d8640-1bc9-44b9-8648-39587534560f.PNG)